src/Core/ProtocolDefines.h

```diff
@@ -30,7 +30,7 @@
 
 #define DBMS_CLUSTER_PROCESSING_PROTOCOL_VERSION 1
 
-#define DBMS_PARALLEL_REPLICAS_PROTOCOL_VERSION 2
+#define DBMS_PARALLEL_REPLICAS_PROTOCOL_VERSION 3
```

Comment:
AFAIS, currently, there is no way to extend the parallel replicas protocol, so replicas with previous version and a new one could work with each other. Regarding the progress bar, it's not that useful, but in general, it could be handy during rolling upgrade